src/components/RegisterForm/majors.json

```diff
@@ -0,0 +1,3 @@
+{
+  "majors": ["Anthropology","Biological Anthropology","Bioengineering","Bioengineering (Bioinformatics)","Bioengineering: BioSystems","Bioengineering (Biotechnology)","Biology with Specialization in Bioinformatics","Ecology, Behavior, and Evolution","General Biology","Human Biology","Microbiology","Molecular and Cell Biology","Neurobiology","Biochemistry and Cell Biology","Molecular Biology","Physiology and Neuroscience","Chemical Engineering","Biochemistry/Chemistry","Chemistry","Environmental Chemistry","Molecular Synthesis","Pharmacological Chemistry","Chinese Studies","Classical Studies","Cognitive Science","Communication","Computer Engineering","Computer Science","Critical Gender Studies","Data Science","Economics","Economics-Public Policy", "Mathematics and Economics","Management Science","Education Sciences","Electrical Engineering","Engineering Physics","Electrical Engineering & Society","Environmental Systems","Ethnic Studies","Public Health","German Studies","Global Health","History","Human Developmental Sciences","Eleanor Roosevelt College Individual Studies","Muir College Special Project Major","Revelle College Individual Major","Thurgood Marshall College Individual Studies","Warren College Individual Studies","Sixth College Individual Studies","International Studies — Anthropology","International Studies — Economics","International Studies — History","International Studies — Linguistics","International Studies — Literature","International Studies — Political Science","International Studies — Sociology","International Studies — Economics","International Studies — Political Science","International Studies — International Business","International Studies — International Business","International Studies — Philosophy","Italian Studies","Japanese Studies","Jewish Studies","Latin American Studies","Latin American Studies with a Concentration in Mexico","Latin American Studies with a Concentration in Migration and Border Studies","Language Studies","Linguistics","Literature/ Writing","Literatures in English","Spanish Literature","World Literature and Culture","Mathematics","Mathematics (Applied)","Mathematics — Applied Science","Mathematics — Computer Science","Probability & Statistics","Aerospace Engineering","Environmental Engineering","Mechanical Engineering","Interdisciplinary Computing and the Arts","Music","Music Humanities","Nanoengineering","Philosophy","General Physics","Physics","Physics-Biophysics","Political Science","Psychology","Business Psychology","Study of Religion","Russian, East European and Eurasian Studies","Earth Sciences","Marine Biology","Oceanic and Atmospheric Sciences","Sociology","Structural Engineering","Dance","Theatre","Third World Studies","Undeclared","Undeclared — Humanities/ Arts","Undeclared — Physical Sciences","Undeclared — Social Sciences","Urban Studies and Planning","Real Estate and Development","Visual Arts","Speculative Design", "Other"]
```

Comment:
Can we do some more condensing here by moving all the specializations of `International Studies: xyz` to just `International Studies`. Same with Undeclared/Bioengineering